README.md

```diff
@@ -1 +1,15 @@
-# hw01
+<hw01>
+
+# My name is *Andrew Bacotti*
+![This is Me!](bondchapel.jpg)
```

Comment:
Since the file "bondchapel.jpg" refers to a local path, other people would not be able to see the photo if they only download the readme.md file. I see that the rendering is fine because now the README.md file and the jpg file are in the same folder in the repo. But the path, https://github.com/ABacotti/hw01/blob/master/bondchapel.jpg?raw=true is the url address for the uploaded photo file.